docs/topics/api/signing.rst

```diff
@@ -36,7 +36,7 @@ validation or review fails.
 If the upload succeeded then it will be submitted for
 validation and you will be able to check its status.
 
-.. http:put:: /api/v3/addons/[string:add-on-id]/versions/[string:version]/
+.. http:put:: /api/v3/addons/[string:addon-id]/versions/[string:version]/
```

Comment:
(Just a joke. I know you are matching the request parameter so it's cool as is.)